devito/mpi/routines.py

```diff
@@ -555,10 +555,10 @@ def _make_sendrecv(self, f, hse, key, msg=None):
         count = reduce(mul, sizes, 1)
         rrecv = Byref(FieldFromPointer(msg._C_field_rrecv, msg))
         rsend = Byref(FieldFromPointer(msg._C_field_rsend, msg))
-        recv = Call('MPI_Irecv', [bufs, count, Macro(dtype_to_mpitype(f.dtype)),
-                                  fromrank, Integer(13), comm, rrecv])
-        send = Call('MPI_Isend', [bufg, count, Macro(dtype_to_mpitype(f.dtype)),
-                                  torank, Integer(13), comm, rsend])
+        recv = IrecvCall('MPI_Irecv', [bufs, count, Macro(dtype_to_mpitype(f.dtype)),
+                                       fromrank, Integer(13), comm, rrecv])
+        send = IsendCall('MPI_Isend', [bufg, count, Macro(dtype_to_mpitype(f.dtype)),
```

Comment:
same story as above

Reply:
I'll push that together with the next changes.